apps/web/src/components/AvatarMap.tsx

```diff
@@ -27,7 +27,7 @@ const AssigneeMap: React.FC<AvatarMapProps> = ({ assignees }) => {
                 size="sm"
                 color="black"
                 bg={
-                  assignee.signed ? '#D1F0D4' : awaiting ? '#AA6B01' : '#E5E5E5'
+                  assignee.signed ? '#D1F0D4' : awaiting ? '#FEF0D8' : '#E5E5E5'
```

Comment:
Nit, but I see the Figma says this hex, but Iris said this hex: #FFF2D9 in the channel in the thread under your PR, do we know which one we should use?